src/ert/scheduler/scheduler.py

```diff
@@ -112,6 +117,19 @@ def __init__(
         self._ee_cert = ee_cert
         self._ee_token = ee_token
         self._publisher_done = asyncio.Event()
+        self._consumer_started = asyncio.Event()
+        self.manifest: Optional[Dict[str, Dict[str, Any]]] = None
```

Comment:
Yes, it makes more sense to have it initially as a `defaultdict()` but the same reasoning as above lead to this implementation. I will refactor and try to avoid it.

Reply:
:+1: